.storybook/docs-root.css

```diff
@@ -426,7 +426,7 @@ h1.fluent {
   letter-spacing: -0.16px;
 }
 
-h1 .fluent-version {
+h1.fluent .fluent-version {
```

Comment:
can you pls point out where is this `.fluent` or `.fluent-version` class being used ? I'm unable to find it in docs = if it's not present lets remove it.

Reply:
The `.fluent` and `.fluent-version` classes are utilized within our codebase.

For example, you can see their usage here: https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/Concepts/Introduction.stories.mdx?plain=1#L9